docs/README.md

```diff
@@ -177,9 +177,15 @@ documentation, please create a thread on the Swift forums under the
   <!-- NOTE: Outdated -->
 - [Lexicon.md](/docs/Lexicon.md):
   Canonical reference for terminology used throughout the project.
-  
+
 ### ABI
 
+- [CallConvSummary.rst](/docs/ABI/CallConvSummary.rst):
+	A concise summary of the calling conventions used for C/C++, Objective-C
+	and Swift on Apple platforms.  Contains references to source documents,
+	where further detail is required.
+- [CallingConvention.rst](/docs/ABI/CallingConvention.md):
+  Describes in detail the Swift calling convention.
```

Comment:
Broken link.

This is already listed under §Surveys, but it makes sense to also list it here.

```suggestion
- [CallingConvention.rst](/docs/ABI/CallingConvention.rst):
  Describes in detail the Swift calling convention.
```

Reply:
Not something I changed, but I'll fix it as it's adjacent to my changes.